ntbs-service/Models/Entities/HospitalDetails.cs

```diff
@@ -19,7 +19,7 @@ public partial class HospitalDetails : ModelBase, IOwnedEntityForAuditing
         [Display(Name = "Consultant")]
         public string Consultant { get; set; }
 
-        [AssertThat("CaseManagerAllowedForTbService", ErrorMessage = ValidationMessages.CaseManagerMustBeAllowedForSelectedTbService)]
+        [AssertThat("ExistingCaseManagerId == null || CaseManagerId == ExistingCaseManagerId || CaseManagerAllowedForTbService", ErrorMessage = ValidationMessages.CaseManagerMustBeAllowedForSelectedTbService)]
```

Comment:
Doesn't this null check mean that if the case doesn't have a case manager then anyone could be added? Could we just get rid of it?

Reply:
Yep good spot. I was surprised none of our tests caught this, so I added one.